bloom-housing/bloom
# 1656/lottery type without explicit date

# Pull Request Template

## Issue

Addresses #1656


## Description

Allows listings to be created as a lottery type without having to know what the lottery date is yet. Changes reviewOrderType from a computed field to an explicit required one, and includes a migration for existing listings to update the reviewOrderType based on the existence of a lottery event.

## Type of change


## How Can This Be Tested/Reviewed?

Create a new listing as FCFS, change it to lottery without a date, and ensure the type stays as lottery. You could also try out the new migration.

## Checklist:
